packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -1,51 +1,42 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import CashierLocked from '../cashier-locked';
-import { useDepositLocked } from '@deriv/hooks';
-import { TRootStore } from 'Types';
+import { useCashierLocked, useDepositLocked } from '@deriv/hooks';
+import { mockStore } from '@deriv/stores';
+import { TRootStore } from '../../../types';
 import CashierProviders from '../../../cashier-providers';
 
 jest.mock('@deriv/hooks', () => ({
     ...jest.requireActual('@deriv/hooks'),
     useDepositLocked: jest.fn(() => false),
+    useCashierLocked: jest.fn(() => false),
 }));
-
-jest.mock('@deriv/hooks', () => ({
-    ...jest.requireActual('@deriv/hooks'),
-    useDepositLocked: jest.fn(() => false),
-}));
-
-jest.mock('@deriv/hooks', () => ({
-    ...jest.requireActual('@deriv/hooks'),
-    useDepositLocked: jest.fn(() => false),
-}));
+const mockUseDepositLocked = useDepositLocked as jest.MockedFunction<typeof useDepositLocked>;
+const mockUseCashierLocked = useCashierLocked as jest.MockedFunction<typeof useCashierLocked>;
 
 describe('<CashierLocked />', () => {
+    beforeEach(() => {
+        mockUseDepositLocked.mockReturnValue(false);
+        mockUseCashierLocked.mockReturnValue(false);
+    });
+
     it('should show the proper message if there is a cryptocashier maintenance', () => {
-        const mockRootStore: DeepPartial<TRootStore> = {
+        const mock_root_store = mockStore({
             client: {
-                account_status: {
-                    cashier_validation: [],
-                },
-                accounts: undefined,
-                loginid: undefined,
+                account_status: { cashier_validation: ['system_maintenance'] },
```

Comment:
Alternatively, we can mock `useIsSystemMaintenance`, but I think this way is easier, yet explicit enough